resources/js/Pages/Results.vue

```diff
@@ -61,6 +61,9 @@
                     </wikit-link>
                 </span>
             </Message>
+            <Message type="notice" v-if="item_ids.length === 0">
```

Comment:
Maybe a comment around this message would make sense? Something like:

```html
<!-- The Results page without item_ids is used by RandomizeController. -->
```

Reply:
ok. i will add it.

Reply:
Done.